apps.gen.go

```diff
@@ -0,0 +1,136 @@
+// Code generated by golang.org/x/tools/cmd/bundle. DO NOT EDIT.
+//   $ bundle -pkg godo -prefix  ./dev/dist/godo
+
+package godo
+
+import ()
+
+// AppDatabaseSpec struct for AppDatabaseSpec
```

Comment:
The comments on the generated file are a bit repetitive. `Foo struct for Foo` Some room for improvement here.

This doesn't need to be blocking, just something to consider for the future.

Reply:
Yup for sure, we're relying on a mix of https://github.com/grpc-ecosystem/grpc-gateway/tree/master/protoc-gen-swagger and https://github.com/OpenAPITools/openapi-generator at the moment for the generation, and that makes some parts of this not as straightforward to modify, but I'll see if there's anything I can do going forward to improve these.